packages/react/src/components/Link/index.css

```diff
@@ -1,3 +1,9 @@
+a.Button--primary,
+a.Button--secondary,
+a.Button--thin {
+  font-weight: var(--font-weight-normal);
+}
+
```

Comment:
I think I may have a better understanding of the confusion on zoom earlier. Anything with the visual appearance of a Link should have a normal font weight. Anything with the visual appearance of the button should have the standard button font weight. Does that help make what's needed here clear?

Based on this, I don't think we need these additional styles.

```suggestion
```

Reply:
That's what this comment (https://github.com/dequelabs/cauldron/pull/1675#discussion_r1761532088) was originally referring to. Sorry for the confusion.

Reply:
Sorry for the back and forth, but this isn't quite still right. 

All components that have a visual appearance of a Button have a font weight of 500:

![image](https://github.com/user-attachments/assets/774daf32-10bb-4472-aaf5-2d8c72d5b4f1)

This is the `<Link variant="Button">`:

![image](https://github.com/user-attachments/assets/5ff861aa-f9ca-45f1-a358-da9daced18d1)

This link button variant has a font weight of 400, which does not match the expectation of what a visual Button should look like. We want to make sure that the visual links have the correct font weight of 400 while visual buttons have the correct font weight of 500.

I believe the change needed here is:

```css
a.Button--primary,
a.Button--secondary,
a.Button--thin {
  font-weight: var(--font-weight-medium);
}
```

The end result should be the only visual changes we should see in screenshots are for components that are visually styled as links.